### DISCUSSION TITLE
Double detection - cat and dog

### DISCUSSION THREAD

**Discussion by bucklevision** [blakeblackshear/frigate#8758] [General]
Hi Folks, I have a 2 camera system with 2 CPU detectors while I get my config up to speed and understand the system. I noticed a strange thing this morning and wondered if it was my config at fault.

I was alerted (about 30 seconds after the event) to a cat walking through my garden. All good, but when I go to the events list, it shows cat and dog events at the same time, with the same clip recorded twice. Do I have a config mistake, should I be running the detector config per-camera somehow? What actually seems to be the happening is the cat is detected at 22 seconds past the hour, and the "dog" (same cat) detected at 32 seconds (according to the thumbnail timestamps) but the same video is used for both.

**Comment by NickM-27** [comment]
This isn't that unusual especially when running frigate 0.12. The model just detected both and thought both were there.

**Reply by bucklevision** [reply]
OK so long as I'm not doing anything wrong that's fine. Odd that it can't keep tracking cat until cat no longer in picture rather than stopping cat and starting dog as if they appear from nowhere but I guess handover from one object to another, or differentiating between in complex scenes is tricky when I think about it. No harm done, just made me smile and question my setup. Ta!
